### ISSUE TITLE
Windows OS can go into sleep mode during playback.

### ISSUE THREAD

**Issue by Canaanabolaanan** [powertab/powertabeditor#235]
May be a bit of a silly point to bring up, I suppose. But clearly sites like Netflix/Youtube don't have issues disabling this function while videos are playing. So I'm guessing there's something this program could communicate to the OS to disable it as well. If so, that'd be swell.

**Comment by Canaanabolaanan** [comment]
I should have specified: Windows 10.

**Comment by cameronwhite** [comment]
Yeah, I can see this being useful when the display is set to turn off after a short amount of idle time. AFAIK Qt doesn't provide anything for this, so we'd have to use the native APIs for each platform.

**Comment by psyomn** [comment]
@cameronwhite Think I can give a jab at this? Might be a fun thing to do in an afternoon. 

@Canaanabolaanan Not sure if you remember me, but I was LJB in the community! Drop me a line if you're still around, trying to see if I can find most powertab members (lost contact with a lot after the forums disappeared).

**Comment by cameronwhite** [comment]
@psyomn - sure, feel free to go ahead with implementing it!

**Comment by Canaanabolaanan** [comment]
@psyomn Hey Simon! How ya doin' man? There is (was?) a facebook group for .ptb members. I lurk here periodically, waiting for either a new computer, or a chromebook release. ;) Happy to see this project develop though. I miss our Note's Art days! I still listen to it. Hope you're doing well. Cheers!
